data_analysis/notebooks/businesses.ipynb

```diff
@@ -3,9 +3,20 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "id": "bf058d3d",
    "metadata": {},
```

Comment:
In 
```python
def mem_data_to_csv(mem_data):
    
    headers = mem_data.keys()
    df = pd.DataFrame(mem_data.items(), columns=headers)
    return df
    df.to_csv ('CNPData.csv', index = False,header=True)
```

the last line is never executed because of the `return` on the previous line.

Reply:
Also I think this returns a dataframe with a single row. mem_data is now a list of dictionaries. Here are 2 options to turn that into a dataframe:
  1. First transform the list dictionaries into a dictionary of lists that can be passed to the `pd.DataFrame` constructor like in [the docs page](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_csv.html)
  2. Loop over the dictionaries, create a single row dataframe or a series for each one, and aggregate them into a single dataframe using [pd.concat](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.concat.html)

Reply:
I think pandas was able to correctly build the dataframe simply by giving it the list of dictionaries. 
I used this and it seems to work.


````
df = pd.DataFrame(mem_data)

df.to_csv ('CNPData.csv', index = False,header=True)
````